cannon/mipsevm/arch/arch32.go

```diff
@@ -61,6 +61,7 @@ const (
 	SysPrlimit64     = 4338
 	SysClose         = 4006
 	SysPread64       = 4200
+	SysStat          = 4106
```

Comment:
question - should be adding tickets to update [the STCannon spec](https://specs.optimism.io/fault-proof/cannon-fault-proof-vm.html)?  I think there are a few changes we probably need to port at this point.

Reply:
Do you recall what those other changes are? From a cursory read of the spec I think `SYS_STAT` is the only one not included.

Reply:
The `fcntl` syscall was modified (added support for `F_GETFD`).  Not sure if there was anything else - that might be it.

Reply:
Synced offline. All that's left is to migrate the 32-bit spec to 64-bit which will is handled by https://github.com/ethereum-optimism/optimism/issues/12253